kratos/utilities/qef_utility.cpp

```diff
@@ -0,0 +1,115 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ariadna Cortes
+//
+
+// Project includes
+#include "qef_utility.h"
+
+namespace Kratos {
+
+    array_1d<double,3> QEF::QEFPoint (
+        const GeometryType& rVoxel,  
+        const GeometryArrayType& rTriangles     
+    ) {
+        array_1d<double,3> x_point;
+        array_1d<double,3> center = CalculateCenter(rVoxel);
+        MatrixType m_center(3,1);
+        column(m_center,0) = center;
+        GeometryArrayType edges = rVoxel.GenerateEdges();
+
+        //Initialize the corresponding matrixes
+        MatrixType ata(3,3,0);  //3x3 matrix initialized to 0
+        MatrixType atb(3,1,0);  //3x1 matrix
+        MatrixType btb(1,1,0);  //1x1 matrix
+
+        for (std::size_t i = 0; i < rTriangles.size(); i++) {
+            array_1d<double,3> normal = CalculateNormal(rTriangles[i]);
+            MatrixType m_normal(3,1);
```

Comment:
We normally use `m` prefix to indicate member variables. If you want to refer to matrix use `mat_` instead.

Reply:
true, done!